docs/content/latest/explore/transactions/explicit-locking.md

```diff
@@ -26,96 +26,85 @@ showAsideToc: true
 
 </ul>
 
-This section explains how explicit locking works in YugabyteDB.
+This section describes how explicit locking works in YugabyteDB.
 
-Explicit row-locks use transaction priorities to ensure that two transactions can never hold conflicting locks on the same row. This is done by the query layer assigning a very high value for the priority of the transaction that is being run to acquire the row lock, causing all other transactions that conflict with the current transaction to fail (because they have a lower value for the transaction priority). YugabyteDB supports most row-level locks, similar to PostgreSQL.
+YugabyteDB supports most row-level locks, similar to PostgreSQL. Explicit row-locks use transaction priorities to ensure that two transactions can never hold conflicting locks on the same row. To do this, the query layer acquires the row lock by assigning a very high value for the priority of the transaction that is being run. This causes all other transactions that conflict with the current transaction to fail, because they have a lower transaction priority.
 
 {{< note title="Note" >}}
 Explicit locking is an area of active development in YugabyteDB. A number of enhancements are planned in this area. Unlike PostgreSQL, YugabyteDB uses optimistic concurrency control and does not block / wait for currently held locks, instead opting to abort the conflicting transaction with a lower priority. Pessimistic concurrency control is currently under development.
 {{</note >}}
 
-The types of row locks currently supported are: 
+The types of row locks currently supported are:
+
 * `FOR UPDATE`
 * `FOR NO KEY UPDATE`
 * `FOR SHARE`
 * `FOR KEY SHARE`
 
-Let us look at en example with the `FOR UPDATE` row lock, where we first `SELECT` a row for update to lock it and subsequently update it. A concurrent transaction should not be able to abort this transaction by updating the value of that row after the row is locked. To try out this scenario, first create an example table with sample data as shown below.
+The following example uses the `FOR UPDATE` row lock. First, a row is selected for update, thereby locking it, and subsequently updated. A concurrent transaction should not be able to abort this transaction by updating the value of that row after the row is locked. 
+
+To try out this scenario, first create an example table with sample data as shown below.
```

Comment:
To try out this scenario, first create an example table with sample data as shown below. -->  To try out this scenario, first create an example table with sample data, as follows: